### ISSUE TITLE
API Protection Policy in AS3 - Feature Request

### ISSUE THREAD

**Issue by jmcalalang** [F5Networks/f5-appsvcs-extension#68]
##### ISSUE TYPE
``` 
- Feature Idea
```
##### AS3 BUILD/ VERSION
```
Feature idea to attached an API Protection policy to an AS3 controlled VIP, this will be a first building block for attaching an APM in a lifecycle with API's
```
##### BIGIP VERSION
```
14.1 support will be needed as this feature is 14.1 introduced
```
##### SUMMARY
```
API Protection was introduced in version 14.1, this allows users to import an OpenAPI spec file and have APM build around this, attaching this created policy in AS3 will allow us to start to build solutions around API protections from a programmatic standpoint.

Starting with this scenario will also allow us to stick API protection into AS3 devices with container services (CIS)
```

@grahamalderson, @MattDierick & @chen23 for visability

**Comment by dstokesf5** [comment]
Thank you for your feedback. It has been passed along to the decision makers.

**Comment by kaustriaf5** [comment]
Tracked internally as 918.

**Comment by tnamba-f5** [comment]
This is an excellent idea, and one that many of my customers would embrace!

**Comment by dstokesf5** [comment]
This issue was lost in an issue tracker migration. I have added it back to our internal product backlog as AUTOTOOL-1965.

**Comment by dstokesf5** [comment]
This was resolved in the 3.26.0 release.
